### PR TITLE
3425 manifest

### PR DESCRIPTION
**Pull request by cakekoa**
# What does this PR do?

Fixes part of #3425.

Add plugin manifest for the RDP exploiter

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [x] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] ~Added relevant unit tests?~
* [ ] ~Do all unit tests pass?~
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: monkey/agent_plugins/exploiters/rdp/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - windows
+title: RDP Exploiter
+version: 1.0.0
+description: Attempts a brute-force attack over RDP using known credentials.
+safe: true
+remediation_suggestion: >-
+    Change user passwords to complex one-use passwords that are not shared with other computers on the network.
+
+    The machine is vulnerable to an RDP attack.
+    An Infection Monkey Agent authenticated over the RDP using stolen/configured credentials.
```
Comment (shreyamalviya) [inline monkey/agent_plugins/exploiters/rdp/manifest.yaml:16]:
```suggestion
    An Infection Monkey Agent authenticated over RDP using stolen/configured credentials.
```

**Review by shreyamalviya — APPROVED** [review]

**Review by ilija-lazoroski — APPROVED** [review]

File: monkey/agent_plugins/exploiters/rdp/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  - linux
+  - windows
+target_operating_systems:
+  - windows
+title: RDP Exploiter
+version: 1.0.0
+description: Attempts a brute-force attack over RDP using known credentials.
+safe: true
+remediation_suggestion: >-
+    Change user passwords to complex one-use passwords that are not shared with other computers on the network.
```
Comment (mssalvatore) [inline monkey/agent_plugins/exploiters/rdp/manifest.yaml:13]: Is it possible to have one-use passwords on Windows?

File: monkey/agent_plugins/exploiters/rdp/manifest.yaml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+    The machine is vulnerable to an RDP attack.
```
Comment (mssalvatore) [inline monkey/agent_plugins/exploiters/rdp/manifest.yaml:15]: This sounds a little dramatic. We were able to login via a remote access protocol. We had correct credentials.

Comment (cakekoa) [reply monkey/agent_plugins/exploiters/rdp/manifest.yaml:15]: Yeah. I dropped [T1210](https://attack.mitre.org/techniques/T1210/) from the tags for this same reason -- we're not _exploiting_ the protocol, just using it.

**Review by mssalvatore — APPROVED** [review]

**Pull request merged**